masterlist.yaml

```diff
@@ -27031,8 +27047,6 @@ plugins:
         util: TES5Edit v3.2.2
   - name: 'Relationship Dialogue Overhaul.esp'
     after:
-      - 'Alternate Start - Live Another Life.esp'
-      - 'Guard Dialogue Overhaul.esp'
```

Comment:
RDO can be loaded before or after these mods, nothing game breaking will happen either way.
So I think adding a hard load after rule is a bit much for these conflicts.